docs/_user_guide/slurm.md

```diff
@@ -66,62 +64,60 @@ python3 yahtzee.py 100000
 ```
 
 The important arguments here are that you request:
+
 * `SBATCH -c` is the number of CPUs
 * `SBATCH -t` is the amount of time for your job
 * `SBATCH --mem` is the amount of total memory
 
-
 Once your slurm script is written, you can submit it to the server by running `sbatch my_submission_script.slurm`.
 
 ## OK - my job is submitted - now what?
 
 You can look at the current job queue by running `squeue`:
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 USER@yen4:~$ squeue
              JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
               1043    normal    a_job    user1 PD       0:00      1 (Resources)
               1042    normal    job_2    user2  R    1:29:53      1 yen11
               1041    normal     bash    user3  R    3:17:08      1 yen11
 ```
 
-Jobs with state (ST) R are running, and PD are pending.  Your job will run based on this queue.
+Jobs with state (ST) R are running, and PD are pending. Your job will run based on this queue.
 
 ## Best Practices
 
 ### Use all of the resources you request
 
-The Slurm scheduler keeps track of the resources you request, and the resources you use. Frequent under-utilization of CPU and Memory will affect your future job priority.  You should be confident that your job will use all of the resources you request.  It's recommended that you run your job on the interactive Yens, and [monitor resource usage](/faqs/howCheckResourceUsage.html) to make an educated guess on resource usage.
+The Slurm scheduler keeps track of the resources you request, and the resources you use. Frequent under-utilization of CPU and Memory will affect your future job priority. You should be confident that your job will use all of the resources you request. It's recommended that you run your job on the interactive Yens, and [monitor resource usage](/_user_guide/best_practices_monitor_usage/){:target="_blank"} to make an educated guess on resource usage.
 
 ### Restructure your job into small tasks
 
-Small jobs start faster than big jobs. Small jobs likely finish faster too.  If your job requires doing the same process many times (i.e. OCR'ing many PDFs), it will benefit you to setup your job as many small jobs.
+Small jobs start faster than big jobs. Small jobs likely finish faster too. If your job requires doing the same process many times (i.e. OCR'ing many PDFs), it will benefit you to setup your job as many small jobs.
 
 ## Tips and Tricks
 
 ### Current Partitions and their limits
```

Comment:
Capitalize all words in titles to be consistent or else

Reply:
So like, "Restructure Your Job into Small Tasks" and "Current Partitions and their Limits"? What about the title "OK - my job is submitted - now what?"